unittests/eosio_system_tester.hpp

```diff
@@ -437,7 +437,7 @@ class eosio_system_tester : public TESTER {
       action act;
       act.account = N(eosio.msig);
       act.name = name;
-      act.data = msig_abi_ser.variant_to_binary( action_type_name, data );
+      act.data = msig_abi_ser.variant_to_binary( action_type_name, data, control->get_abi_serializer_max_time_ms() );
```

Comment:
get_abi_serializer_max_time_ms of chain_controller is already removed (this piece of code is actually just inside a comment)